### PR TITLE
fix(SFS): fix sfs lint error

### PR DESCRIPTION
**Pull request by wanglong3710**
**What this PR does / why we need it**:
fix sfs lint error

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [X] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/sfs/' TESTARGS='-run TestAccSFSAccessRuleV2_basic'
```
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/sfs/ -v -run TestAccSFSAccessRuleV2_basic -timeout 360m -parallel 4
=== RUN   TestAccSFSAccessRuleV2_basic
=== PAUSE TestAccSFSAccessRuleV2_basic
=== CONT  TestAccSFSAccessRuleV2_basic
--- PASS: TestAccSFSAccessRuleV2_basic (102.52s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/sfs       102.566s

```
make testacc TEST='./huaweicloud/services/acceptance/sfs/' TESTARGS='-run TestAccSFSFileSystemV2_basic'
```
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/sfs/ -v -run TestAccSFSFileSystemV2_basic -timeout 360m -parallel 4 
=== RUN   TestAccSFSFileSystemV2_basic 
=== PAUSE TestAccSFSFileSystemV2_basic
=== CONT  TestAccSFSFileSystemV2_basic
--- PASS: TestAccSFSFileSystemV2_basic (100.95s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/sfs       100.995s

```
make testacc TEST='./huaweicloud/services/acceptance/sfs/' TESTARGS='-run TestAccSFSFileSystemV2DataSource_basic'
```
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/sfs/ -v -run TestAccSFSFileSystemV2DataSource_basic -timeout 360m -parallel 4 
=== RUN   TestAccSFSFileSystemV2DataSource_basic 
=== PAUSE TestAccSFSFileSystemV2DataSource_basic
=== CONT  TestAccSFSFileSystemV2DataSource_basic
--- PASS: TestAccSFSFileSystemV2DataSource_basic (65.24s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/sfs       65.282s

```
make testacc TEST='./huaweicloud/services/acceptance/sfs/' TESTARGS='-run TestAccTurbosDataSource_basic'
```
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/sfs/ -v -run TestAccTurbosDataSource_basic -timeout 360m -parallel 4 
=== RUN   TestAccTurbosDataSource_basic 
=== PAUSE TestAccTurbosDataSource_basic
=== CONT  TestAccTurbosDataSource_basic
--- PASS: TestAccTurbosDataSource_basic (441.37s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/sfs       441.407s
```

### CODE REVIEW

File: huaweicloud/services/sfs/resource_huaweicloud_sfs_file_system.go [hunk 1]
```diff
@@ -448,13 +456,13 @@ func waitForSFSFileRefresh(sfsClient *golangsdk.ServiceClient, shareID string) r
...
 
-func resourceSFSMetadataV2(d *schema.ResourceData, config *config.Config) map[string]string {
+func resourceSFSMetadataV2(d *schema.ResourceData, cfg *config.Config) map[string]string {
 	m := make(map[string]string)
 	for key, val := range d.Get("metadata").(map[string]interface{}) {
 		m[key] = val.(string)
 	}
 
-	epsID := common.GetEnterpriseProjectID(d, config)
+	epsID := common.GetEnterpriseProjectID(d, cfg)
```
Comment (deer-hang) [inline huaweicloud/services/sfs/resource_huaweicloud_sfs_file_system.go]:
```suggestion
	epsID := cfg.GetEnterpriseProjectID(d)
```

**Comment by wanglong3710** [comment]
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/sfs/ -v -run TestAccSFSFileSystemV2_basic -timeout 360m -parallel 4 
=== RUN   TestAccSFSFileSystemV2_basic 
=== PAUSE TestAccSFSFileSystemV2_basic
=== CONT  TestAccSFSFileSystemV2_basic
--- PASS: TestAccSFSFileSystemV2_basic (123.56s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/sfs       123.596s

File: huaweicloud/services/sfs/resource_huaweicloud_sfs_file_system.go [hunk 2]
```diff
@@ -348,7 +354,8 @@ func resourceSFSFileSystemV2Update(ctx context.Context, d *schema.ResourceData,
 			if deny.Err != nil {
 				return diag.Errorf("error changing access rules for SFS file system: %s", deny.Err)
 			}
-			d.Set("share_access_id", "")
+			mErr := multierror.Append(nil, d.Set("share_access_id", ""))
+			return diag.FromErr(mErr.ErrorOrNil())
```
Comment (ShiChangkuo) [inline huaweicloud/services/sfs/resource_huaweicloud_sfs_file_system.go]: we should not return here as other parameters maybe changed.

**Comment by wanglong3710** [comment]
make testacc TEST='./huaweicloud/services/acceptance/sfs/' TESTARGS='-run TestAccSFSFileSystemV2_basic'
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/sfs/ -v -run TestAccSFSFileSystemV2_basic -timeout 360m -parallel 4 
=== RUN   TestAccSFSFileSystemV2_basic 
=== PAUSE TestAccSFSFileSystemV2_basic
=== CONT  TestAccSFSFileSystemV2_basic
--- PASS: TestAccSFSFileSystemV2_basic (118.11s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/sfs       118.161s

make testacc TEST='./huaweicloud/services/acceptance/sfs/' TESTARGS='-run TestAccSFSAccessRuleV2_basic'
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/sfs/ -v -run TestAccSFSAccessRuleV2_basic -timeout 360m -parallel 4 
=== RUN   TestAccSFSAccessRuleV2_basic 
=== PAUSE TestAccSFSAccessRuleV2_basic
=== CONT  TestAccSFSAccessRuleV2_basic
--- PASS: TestAccSFSAccessRuleV2_basic (105.49s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/sfs       105.538s

**Pull request merged**